src/pages/faq.md

```diff
@@ -6,76 +6,56 @@ metaDescription: "Frequently Asked Questions about Qri"
-
-The longer story is the data you add to qri lives on your computer, and is made available on the distributed web.
-
+Data on qri lives on your hard drive in an IPFS repository, and is made available on the distributed web. Learn more about [how Qri leverages IPFS here](https://qri.io/docs/reference/ipfs_to_qri#qri-stores-datasets-in-ipfs). 
 
 ### Do you consider the Dat project an alternative to IPFS?
 
 We reviewed both Dat and IPFS white papers at the same time, and keyed in on one major feature of Dat which made it incompatible with the scope and goals we had for Qri out of the gate.
 
-Dat, like git, uses a repository model. So, if a user were to create a dataset containing a certain csv file, and another user had the exact same csv file (because, say, they were both downloaded from the same URL around the same time), the files would not ‘collide’ - meaning both physical files would not deduplicate, and storage costs (and all other complications) would scale as more datasets are added to the network by other users.
+Dat, like git, uses a repository model. So, if a user were to create a dataset containing a certain csv file, and another user had the exact same csv file (because, say, they were both downloaded from the same URL around the same time), the files would not ‘collide’ — meaning both physical files would not deduplicate, and storage costs (and all other complications) would scale as more datasets are added to the network by other users.
 
 In an ideal data commons, identical datasets would collide and deduplicate to a single file on one’s hard drive. That doesn’t happen with Dat, BitTorrent or several others. The only technology that has that key property is IPFS, because of the ‘single swarm’ style of content propagation.
 
 We’re really impressed with the Dat community and the aims of the project as a whole, and eventually we’d like to be able to support exporting to Dat from qri.
 
-
-### What kinds of data does qri support?
-
-ALL kinds of data! We're rapidly building in support for every and any kind of data. Today, qri works best with CSV & JSON files - you can put all kinds of data in CSV & JSON files.
-
+### Which data formats does Qri support? How does qri support / get along with some of the more obscure dataset formats?
+ 
+Qri supports CSV, JSON, XSLX and CBOR formats. Much of our tooling and GUIs are optimized for tabular data (CSVs), but the other formats can be used with Qri CLI. 
 
 ### Who can see my data?
 
-Currently **all data on qri is public.** We’re working on support for both encrypted data and private networks. You can track the progress of encryption and private networks feel free to +1 any of those comments to let us know you want them faster.
+Currently all datasets on qri are public. By publishing a dataset, you make it discoverable on Qri.cloud. Unpublished datasets are still discoverable by other qri users, but it’s more akin to having an unlisted phone number. 
```

Comment:
All datasets on qri.cloud are public.  You can still have datasets locally that aren't visible by anyone.  We should clarify this point.